notebooks/community/feature_store/vertex_ai_feature_store_based_llm_grounding_tutorial.ipynb

```diff
@@ -0,0 +1,1236 @@
+{
```

Comment:
<p>Update the license year to 2024.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2652/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>